### PR TITLE
ref(scss) Variables cleanup

### PR DESCRIPTION
**Pull request by robertpin**
Remove some unused variables
Replace variables that are only used once with their value

### CODE REVIEW

File: css/_base.scss [hunk 1]
```diff
@@ -31,8 +31,8 @@ body {
     font-size: 12px;
     font-weight: 400;
     overflow: hidden;
-    color: $defaultColor;
-    background: $defaultBackground;
+    color: #F1F1F1;
+    background: #040404;
```
Comment (saghul) [inline css/_base.scss:35]: Leave comment indicating this needs to match the value in interface_config pl.

Comment (saghul) [reply css/_base.scss:35]: Wouldn't it make sense to have descriptive variables for these? Like `bodyBackground` etc?

**Review by saghul — APPROVED** [review]
LGTM with a question.

File: css/_base.scss (hunk 1 above)
Comment (robertpin) [reply css/_base.scss:35]: Considering it's only used here I don't see the benefit of adding a SCSS variable

Comment (robertpin) [reply css/_base.scss]: Ideally, we'd move everything to JSS so it would use the theme colors

**Comment by robertpin** [comment]
Jenkins test this please

File: css/_base.scss (hunk 1 above)
Comment (saghul) [reply css/_base.scss]: Makes sense.

**Comment by robertpin** [comment]
Test failure is unrelated

**Comment by robertpin** [comment]
Jenkins test this please

**Review by saghul — APPROVED** [review]

**Comment by robertpin** [comment]
Test failure is unrelated

**Pull request merged**